lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -346,54 +249,83 @@ Here are just a few examples of summary data being used for important things:
 - Adults are often summarized by a single number -- like their credit score -- which determines their ability to get a job or a home loan.
 - When buying uniforms for a sports team, a coach might look for the most common size that the players wear.
 
-@lesson-instruction{
-What other examples can you think of where a number or two are used to summarize something complex?
-}
+
+We are going to learn how to use @proglang to quickly, easily compute the three different measures of center. That way, we can avoid number crunching and spend our energy thoughtfully deciding *which* measure of center is the most appropriate in a given situation.
+
 
 === Investigate
 
-You now have three different ways to measure center in a dataset. Every kind of summary has situations in which it does a good job of reporting what’s typical, and others where it doesn’t really do justice to the data. 
 
-But how do you know which one to use? Depending on the shape of the dataset, a measure could be really useful or totally misleading! 
+@ifproglang{pyret}{
 
-@slidebreak
+Pyret has functions that will compute mean, median, and mode.
 
-@QandA{
-@Q{"In 2003, the average American family earned $43,000 a year -- well above the poverty line! Therefore, very few Americans were living in poverty."} 
-@Q{Do you trust this statement? Why or why not?}
-@A{Sample response: The mean is sensitive to outliers, and billionaires like Elon Musk, Jeff Bezos, etc. pull the mean heavily to the right. This makes it appear that the "average" American family earns far more than they actually do. That's why the conclusion "very few Americans were living in poverty" cannot be drawn based on the mean.}
+@hspace{2em} @show{ (contract "mean" '("Table" "String") "Number" )}
+
+@hspace{2em} @show{ (contract "median" '("Table" "String") "Number" )}
+
+@hspace{2em} @show{ (contract "modes" '("Table" "String") "List" )}
+
+Note: `List` is a new data type!
 }
 
+
+@ifproglang{codap}{
+To compute the mean and median in CODAP, create a graph of randomly distributed points, then drag a quantitative column to the x-axis. From the `Measure` menu, select Mean or Median. *If this information is not on your Data Visualizations Organizer, add it now!*
+}
+
+
+@lesson-instruction{
+- Open the @starter-file{animals} in @proglang.
+- Complete @printable-exercise{summarizing-columns-moc.adoc}, using @proglang to compute and record all three measures of center for the `pounds` column.
+- Once you have completed the table, respond to the remaining questions using the information you have recorded on the table.
```

Comment:
What table? The previous instructions don't mention a table. Unless a kid is already open to the right page and looking at it, this will be confusing